frame/crowdloan-rewards/src/lib.rs

```diff
@@ -0,0 +1,325 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
may be add some doc about pallet, either into readme or //! module comment or into description attribute in toml, may be just link to wiki